### PR TITLE
NVLI-214 changes the permission for anynoums user

### PR DESCRIPTION
**Pull request by dishabhadra**
@piyuesh23 : Do the code review.

I change the permission of Search Block so anonymous user can acces.

### CODE REVIEW

File: docroot/modules/custom/nvli_custom_search/src/Plugin/Block/NewSearchBlock.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ class NewSearchBlock extends BlockBase {
    * {@inheritdoc}
    */
   protected function blockAccess(AccountInterface $account) {
```
Comment (piyuesh23) [inline docroot/modules/custom/nvli_custom_search/src/Plugin/Block/NewSearchBlock.php:27]: @dishabhadra What is this block intended for? The name NewSearchBlock doesn't give any insight about it.

Comment (dishabhadra) [inline docroot/modules/custom/nvli_custom_search/src/Plugin/Block/NewSearchBlock.php:27]: @piyuesh23 : This block will render the search form.
This file I already pushed but now I change the permission.

Comment (piyuesh23) [inline docroot/modules/custom/nvli_custom_search/src/Plugin/Block/NewSearchBlock.php:27]: @dishabhadra Can we change the Class name to something more semantic?

Comment (dishabhadra) [inline docroot/modules/custom/nvli_custom_search/src/Plugin/Block/NewSearchBlock.php:27]: @piyuesh23 : Changes are done.

**Pull request merged**